.idea/runConfigurations.xml

```diff
@@ -1,3 +1,14 @@
+<<<<<<< HEAD
```

Comment:
I think you should remove these merge artifacts.

Reply:
True ! It is weird that it didn’t tell me that I left it. I’ll probably remove the modification anyway since I don’t know why it has been modified.